### ISSUE TITLE
Static "transform" method

### ISSUE THREAD

**Issue by ivanocortesini** [HotelsDotCom/bull#25]
Add static "transform" method to BeanUtils class like:
```
public static <T, R> Function<? extends T, R> getTransformer(Class<R> toBeanClass) {
    return fromBean -> new BeanUtils().getTransformer().transform(fromBean, toBeanClass);
}
```
This will give the possibility to use your library with streams like:
```
List<FromType> sourceList = ...

List<ToType> destinationList = sourceList.stream()
      .map(BeanUtils.getTransformer(ToType.class))
      .collect(Collectors.toList());
```

**Comment by fborriello** [comment]
Good idea @ivanocortesini, thanks for your suggestion, we have added it to our activities backlog, we will keep you posted. 
However, if you want to contribute directly, you are more than welcome.
Thanks

**Comment by fborriello** [comment]
Good morning Ivano, feature implemented. It's available in the release 1.1.3.
Thanks for contributing

**Comment by ivanocortesini** [comment]
Thank you Fabio.
